taskcluster/docker/linux/Dockerfile

```diff
@@ -88,8 +88,9 @@ RUN apt-get update -qq \
 
 RUN pip3 install --upgrade pip
 RUN pip3 install \
-    'taskcluster>=4,<5' \
-    pyyaml
+    pyyaml \
+    taskcluster \
```

Comment:
Taking advantage to unpin this. I've triggered a staging release to ensure this still works - https://firefox-ci-tc.services.mozilla.com/tasks/groups/NfxLsxqWTLKNAvVgYsZKcQ